pom.xml

```diff
@@ -8,10 +8,16 @@
   </parent>
 
   <artifactId>activiti-cloud-runtime-bundle-service-parent</artifactId>
+  <version>7-201710-EA-elias_1501_listener_scope-SNAPSHOT</version>
   <name>Activiti Cloud :: Runtime Bundle Service Parent</name>
   <description>Activiti Cloud :: Runtime Bundle Service Parent</description>
   <packaging>pom</packaging>
 
+  <properties>
+    <version.activiti>7-201710-EA-SNAPSHOT</version.activiti>
```

Comment:
Not sure that we want to keep this change. It was necessary here to depend on `7-201710-EA-SNAPSHOT` instead of `7-201710-EA-elias_1501_listener_scope-SNAPSHOT`

Reply:
yes I think it is fine.. I will merge this PR now